.gitignore

```
@@ -34,5 +34,7 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+.env
 
-.env
\ No newline at end of file
+api/data
```

Maybe create a generic sample data folder?

```suggestion
api/sample-data/*
```